docs/web-analytics.md

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
I noticed a problem in the way the documentation is brought to the reader. You are explaining how to configure the sandbox project when you should in my opinion explain how to set-up a richie project from scratch. If you look at the existing configuration, it is written to explain which settings should be present in Django and not how to do that. For the sandbox and our own project, we use [Django Configurations](https://github.com/jazzband/django-configurations) but other developpers may choose to use a simple `settings.py` file directly...

So I think we could ask them to set a `WEB_ANALYTICS_ID` and a `WEB_ANALYTICS_LOCATION` setting (however they want) and derive all the code on these 2 settings. Then in the sandbox we use DjangoConfigurations to set these 2 settings as an example... what do you think?

Reply:
Ok. I'm going to change the docs.
But I think `WEB_ANALYTICS_GOOGLE_ANALYTICS_ID` is a better name.

Reply:
Done.
Now it only has this 2 settings: `WEB_ANALYTICS_GOOGLE_ANALYTICS_ID` and `WEB_ANALYTICS_LOCATION` and on sandbox the environment variables has the same name.
Also reviewed the docs with your feedback.